webv2/table/rename_column.go

```diff
@@ -22,19 +22,12 @@ import (
 // renameColumn renames given column to newname and update in schema.
 func renameColumn(newName, table, colName string, conv *internal.Conv) {
 
-	sp := conv.SpSchema[table]
-
 	renameColumnNameTableSchema(conv, table, colName, newName)
 
-	// update foreignKey relationship Table.
-	for _, fk := range sp.Fks {
-		renameColumnNameTableSchema(conv, fk.ReferTable, colName, newName)
-	}
-
-	for _, sp := range conv.SpSchema {
+	for k, sp := range conv.SpSchema {
 		for j := 0; j < len(sp.Fks); j++ {
 			if sp.Fks[j].ReferTable == table {
-				renameColumnNameTableSchema(conv, sp.Name, colName, newName)
+				renameColumnNameInSpannerForeignkeyReferColumns(conv.SpSchema[k], colName, newName)
```

Comment:
Interleave suggestions are not getting updated based on change in column name , like 2nd and 3rd one here: https://drive.google.com/file/d/1C7RRXbPwDtp5-PR_y4gzT3diA3Yt3H01/view?usp=sharing&resourcekey=0-rVhn-gDS2wHDKlbbJiJfLA